examples/log_stream.rs

```diff
@@ -26,10 +26,10 @@ async fn main() -> Result<()> {
             ..LogParams::default()
         })
         .await?
-        .boxed();
+        .lines();
 
     while let Some(line) = logs.try_next().await? {
-        info!("{:?}", String::from_utf8_lossy(&line));
+        info!("{}", line);
```

Comment:
Like how this cleaned up the example 👍

Reply:
@aryan9600 @clux  The previous implementation was compatible with tokio-streams so that we can specify a [timeout for the stream](https://github.com/kdash-rs/kdash/blob/main/src/network/stream.rs#L109), but switching to `AsyncBufReadExt` breaks that. Any pointer on how to get the timeout back?

Reply:
nevermind, it works if I add `futures` as a dependency nd import `use futures::AsyncBufReadExt;`